File: Python/Ex_20.py
score = []

for c in range(1, 3):
    s = float(input(f'Enter the P{c} score: '))
    score.append(s)

average = (score[0] + 2 * score[1]) / 3

if average >= 5.0: print("Approved!")

else: print("Disapproved")

File: c/Ex_20.c
#include <stdio.h>

int main(int argc, char const *argv[])
{
    float p1, p2, average;

    printf("Enter the P1 score: ");
    scanf("%f", &p1);
    printf("Enter the P2 score: ");
    scanf("%f", &p2);

    average = (p1 + p2 * 2) / 3;

    if(average >= 5)
    {
        printf("Approved!");
    }
    else
    {
        printf("Disapproved");
    }
    return 0;
}
